src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/FhirServerBuilderEverythingOperationRegistrationExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Features.Operations.Everything;
+using Microsoft.Health.Fhir.Core.Registration;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class FhirServerBuilderEverythingOperationRegistrationExtensions
+    {
+        public static IFhirServerBuilder AddEverythingOperation(this IFhirServerBuilder fhirServerBuilder)
```

Comment:
>AddEverythingOperation [](http://example.com/codeflow?start=41&length=22)

Considering you aiming to have multiple operations, maybe it's worth to call  AddEverythingOperation`s` ?

Reply:
Sure! It is fixed in the update.